scripts/Diffraction/isis_powder/gem_routines/gem_calibration_algs.py

```diff
@@ -138,7 +138,9 @@ def _calibration_processing(
         InputWorkspace=cross_correlated, GroupingFileName=offset_file, OutputWorkspace=offsets_ws_name, **get_det_offset_params
     )
     rebinned_tof = mantid.ConvertUnits(InputWorkspace=rebinned, Target="TOF")
-    aligned = mantid.AlignDetectors(InputWorkspace=rebinned_tof, CalibrationFile=offset_file)
+    mantid.ApplyDiffCal(InstrumentWorkspace="rebinned_tof", CalibrationFile=offset_file)
```

Comment:
Here it would actually be better to use the offsets workspace (`OffsetsWorkspace=offset_ws_name` here). The act of writing to a fixed width column can truncate a number, which in this case can lead to quite large deviations (relative to typical absolute tolerances used in system tests) because the y-values are large and the peak is quite sharp.
<img width="900" alt="image" src="https://github.com/user-attachments/assets/ae78bb5b-8997-4e4e-b309-29a2830d588f" />
Even though the d-spacing of the workspaces calibrated using the file and the offsets workspace differ only by ~1e-8

Reply:
I have kept the use of the offsets file so as to 'fix' the test and preserve the behaviour - in practice this difference in offset used will never be noticed in the refinement, it is only noticeable in system tests.